SessionUIKit/Screens/Shared/SessionListScreen+ListItemView.swift

```diff
@@ -0,0 +1,361 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import SwiftUI
+import DifferenceKit
+
+// MARK: - ListItemCell
+
+public struct ListItemCell: View {
```

Comment:
Similar to the previous comment - splitting these UI elements into separate files will make it easier to tweak, preview and add new elements in the future